community.md

```diff
@@ -50,31 +50,30 @@ We run an announcement-only, no-marketing-spam mailing list that we will use to
 
 ## Want to report issues? 
 
-Oh no, you hit a bump on the road! Please tell us - we want to fix all bugs!
+Oh no, you hit a bump on the road? Please tell us - we want to fix all bugs!
 
-- Found a bug in the compiler, standard libraries or the runtime? Create an issue here: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">https://github.com/ballerina-platform/ballerina-lang/issues</a>
-- Found a bug in the language specification? Create an issue here: <a href="https://github.com/ballerina-platform/ballerina-spec/issues">https://github.com/ballerina-platform/ballerina-spec/issues</a>
-- Found a bug on the website? Create an issue here: <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">https://github.com/ballerina-platform/ballerina-dev-website/issues</a> 
+- Found a bug in the compiler, runtime, standard library, or tooling? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
+- Found a bug in the language specification? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
+- Found a bug on the website? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo. 

```

Comment:
IMO we need to include details on how to report security issues here also by linking ballerina security policy.

Reply:
Added as follows:

`Found a security flaw? Send an email to security@ballerina.io. For details, see the <a href="https://ballerina.io/security/">security policy</a>.`